.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-.env.*
\ No newline at end of file
+.env.development
+.env.test
```

Comment:
You’ve managed to push the .gitignore file here! It seems to be because you’ve changed .env.* to lines 2 and 3. Please delete these and replace them with .env.* which will prevent this happening! +1